webpack/assets/javascripts/react_app/components/common/Slot/__snapshots__/Slot.test.js.snap

```diff
@@ -193,17 +193,6 @@ exports[`Slot-Fill render multiple fills 1`] = `
 </Slot>
 `;
 
-exports[`Slot-Fill render multiple fills 2`] = `
-Object {
-  "extendable": Object {
-    "slot-1": Object {
-      "some-key-1": 100,
-      "some-key-2": 200,
-    },
-  },
-}
-`;
-
```

Comment:
I think there is some sort of unique name issue that jest now enforces, i.e. you can no longer have `Slot-Fill render multiple fills 1` and `Slot-Fill render multiple fills 2`